lambda/shared/person.go

```diff
@@ -0,0 +1,40 @@
+package shared
+
+type Address struct {
+	Line1    string `json:"line1" dynamodbav:""`
+	Line2    string `json:"line2" dynamodbav:""`
+	Line3    string `json:"line3" dynamodbav:""`
+	Town     string `json:"town" dynamodbav:""`
+	Postcode string `json:"postcode" dynamodbav:""`
+	Country  string `json:"country" dynamodbav:""`
+}
+
+type Person struct {
+	FirstNames  string  `json:"firstNames" dynamodbav:""`
+	Surname     string  `json:"surname" dynamodbav:""`
+	DateOfBirth Date    `json:"dateOfBirth" dynamodbav:""`
+	Email       string  `json:"email" dynamodbav:""`
+	Address     Address `json:"address" dynamodbav:""`
+}
+
+type Donor struct {
+	Person
+	OtherNames string `json:"otherNames" dynamodbav:""`
+}
+
+type AttorneyStatus string
+
+const (
+	AttorneyStatusActive      = AttorneyStatus("active")
+	AttorneyStatusReplacement = AttorneyStatus("replacement")
+	AttorneyStatusRemoved     = AttorneyStatus("removed")
```

Comment:
Should inactive also be included here?

Reply:
We haven't agreed on this yet, but I'm using "removed" as a replacement for "inactive" to avoid confusion with replacement attorneys (attorneys who were active vs attorneys who will be active)